mix.exs

```diff
@@ -47,11 +47,11 @@ defmodule PolymorphicEmbed.MixProject do
 
   defp deps do
     [
-      {:ecto, "~> 3.9"},
+      {:ecto, "~> 3.11"},
```

Comment:
Not sure if this warning was present before, but we have a compilation warning in test;

```
warning: passing a MFA to :with in cast_assoc/cast_embed is deprecated, please pass an anonymous function instead
  (ecto 3.11.1) lib/ecto/changeset/relation.ex:119: Ecto.Changeset.Relation.do_cast/7
  (ecto 3.11.1) lib/ecto/changeset/relation.ex:344: Ecto.Changeset.Relation.single_change/5
  (ecto 3.11.1) lib/ecto/changeset/relation.ex:112: Ecto.Changeset.Relation.cast/5
  (ecto 3.11.1) lib/ecto/changeset.ex:1278: Ecto.Changeset.cast_relation/4
  (polymorphic_embed 3.0.5) test/support/models/not_polymorphic/reminder.ex:30: PolymorphicEmbed.Regular.Reminder.custom_changeset/2
  test/polymorphic_embed_test.exs:705: anonymous fn/2 in PolymorphicEmbedTest."test custom changeset by passing MFA"/1
  (elixir 1.16.0) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
  test/polymorphic_embed_test.exs:686: PolymorphicEmbedTest."test custom changeset by passing MFA"/1
  (ex_unit 1.16.0) lib/ex_unit/runner.ex:472: ExUnit.Runner.exec_test/2
  (stdlib 5.0.2) timer.erl:270: :timer.tc/2
  (ex_unit 1.16.0) lib/ex_unit/runner.ex:394: anonymous fn/6 in ExUnit.Runner.spawn_test_monitor/4
```

Can be fixed with this patch

```patch
diff --git a/test/support/models/not_polymorphic/reminder.ex b/test/support/models/not_polymorphic/reminder.ex
index 827fb74..f33c7cf 100644
--- a/test/support/models/not_polymorphic/reminder.ex
+++ b/test/support/models/not_polymorphic/reminder.ex
@@ -28,7 +28,7 @@ defmodule PolymorphicEmbed.Regular.Reminder do
     struct
     |> cast(values, [:date, :text])
     |> cast_embed(:channel,
-      with: {PolymorphicEmbed.Regular.Channel.SMS, :custom_changeset, ["foo", "bar"]}
+      with: &PolymorphicEmbed.Regular.Channel.SMS.custom_changeset(&1, &2, "foo", "bar")
     )
     |> validate_required(:date)
   end
```

Reply:
looks like it's new in 3.10. will address here!

Reply:
looks like this is the point of the test (`"test custom changeset by passing MFA"`), so i'll just remove the test and reorganize in a follow-up MR.